packages/amplify-cli/src/lib/packageManagerHelpers.js

```diff
@@ -0,0 +1,44 @@
+const path = require('path');
+const fs = require('fs');
+
+/**
+ * Determine the package manager of the current project
+ *
+ * @return {string} 'yarn' if yarn.lock exists, 'npm' if package.json exists, undefined otherwise
+*/
+async function getPackageManager() {
+    const yarnLock = './yarn.lock';
```

Comment:
Should the sample app decide what package manager to be used. Its possible that the sample app used yarn to install the dependencies, but the customer who is cloning this project does not even have yarn installed. We should also check if the customer has this package manage installed before running it

Reply:
Added a check to see if the user has yarn installed